cli/azd/pkg/devcenter/manager.go

```diff
@@ -265,15 +271,18 @@ func (m *manager) LatestArmDeployment(
 		tagEnvTypeName, envTypeOk := d.Tags[DeploymentTagEnvironmentType]
 		tagEnvName, envOk := d.Tags[DeploymentTagEnvironmentName]
 
-		if !devCenterOk || !projectOk || !envTypeOk || !envOk {
-			return false
-		}
+		// ARM runner deployments contain the deployment tags for the specific environment
+		isArmDeployment := devCenterOk && strings.EqualFold(*tagDevCenterName, m.config.Name) &&
+			projectOk && strings.EqualFold(*tagProjectName, m.config.Project) &&
+			envTypeOk && strings.EqualFold(*tagEnvTypeName, m.config.EnvironmentType) &&
+			envOk && strings.EqualFold(*tagEnvName, env.Name)
 
-		if *tagDevCenterName == m.config.Name ||
-			*tagProjectName == m.config.Project ||
-			*tagEnvTypeName == m.config.EnvironmentType ||
-			*tagEnvName == env.Name {
+		// Support for untagged Bicep ADE deployments
+		// If the deployment is not tagged but starts with the current date and is running
+		// this is another indication that this is the latest running Bicep deployment
```

Comment:
To find the current running deployment used for progress display it will be the current date.  For env refresh scenarios it will be a previous date.

The deployment list gets sorted by date descending so this will find the most recent deployment that matches the regex pattern.

Reply:
That makes total sense!

Can you help confirm if this comment is up-to-date based on that implementation? It feels like we're just matching on a date regex, not on the current date:

```
// If the deployment is not tagged but starts with the current date and is running
// this is another indication that this is the latest running Bicep deployment
```